geometry/SettingsViewController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  SettingsViewController.swift
+//  geometry
+//
+//  Created by Dmitry Tolmachev on 21.02.16.
+//  Copyright © 2016 Al Digit. All rights reserved.
+//
+
+import UIKit
+
+protocol SettingsViewControllerDelegate: class {
+    func settingsViewControllerFinished(settingsViewController: SettingsViewController)
+}
+
+class SettingsViewController: UIViewController
+{
+    var red: CGFloat = 0.0
+    var green: CGFloat = 0.0
+    var blue: CGFloat = 0.0
+    weak var delegate: SettingsViewControllerDelegate?
+    
+    @IBAction func close(sender: AnyObject) {
+        dismissViewControllerAnimated(true, completion: nil)
+        self.delegate?.settingsViewControllerFinished(self)
+    }
+    
+    @IBOutlet weak var redcolor: UISlider!
+    @IBOutlet weak var greencolor: UISlider!
+    @IBOutlet weak var bluecolor: UISlider! 
+    
+    @IBOutlet weak var labelRed: UILabel!
+    @IBOutlet weak var labelGreen: UILabel!
+    @IBOutlet weak var labelBlue: UILabel!
+    
+    @IBAction func colorChanged(sender: UISlider) {
+        red = CGFloat(redcolor.value / 255.0)
```

Comment:
Casting type is usually unwanted and potentially error prone. If var red = 0.0 is used above, this type cast won't be needed.